### DISCUSSION TITLE
MCDU never displays CRZ when cruising.

### DISCUSSION THREAD

**Discussion by andreda** [flybywiresim/aircraft#3018] [Q&A] [answered]
My MCDU never displays CRZ. 

Am I missing something?

![image](https://user-images.githubusercontent.com/19509499/104413601-96b86e00-553c-11eb-9d4d-e844fde10bc7.png)

**Comment by Inaxair** [comment, marked as answer]
Hello @andreda 

You'll need to go in the flight phase, and select previous phase. This should display the CRZ phase on the MCDU.

Best regards,
Inaxair from FlyByWire Simulations

**Comment by andreda** [comment]
Looks like I am rusty.. would you please recommend a good MCDU manual/course? I'm outdated.. not flying FS since P3D2

**Comment by Inaxair** [comment]
Hello @andreda 

I will recommend few documentations:

Airline2Sim A320 Cadet Training program, a great course about everything for the A320. However it is payware, but it's worth it!
https://www.airline2sim.com/products/airbus-a320-cadet-training-program

Another alternative would be this video, showing how to setup a MCDU: https://www.youtube.com/watch?v=c7UJlJMJdfk

I hope I helped you!
Inaxair from FlyByWire Simulations

**Reply by andreda** [reply]
Thank you! Super helpful!

I will check that... however I am still stuck with this CLB message...

I am at cruise altitude...the MCDU switched to CRZ..

I am pressing the NEXT PHASE

![image](https://user-images.githubusercontent.com/19509499/104551037-13fae600-5604-11eb-9ccf-61f5c6d976e3.png)

Then pressing the PREV PHASE...
![image](https://user-images.githubusercontent.com/19509499/104551019-07768d80-5604-11eb-8687-580318e524c2.png)

The ECAM message doesn't change :(
